### PR TITLE
Simplify output from `make pyspec`

### PR DESCRIPTION
**Pull request by jtraglia**
From this:

image

To this:

image

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -394,8 +404,6 @@ def initialize_options(self):
     def finalize_options(self):
         """Post-process options."""
         if len(self.md_doc_paths) == 0:
-            print("no paths were specified, using default markdown file paths for pyspec"
```
Comment (ralexstokes) [inline setup.py:397]: does this PR change anything if someone does provide non-default paths?

File: setup.py [hunk 2]
```diff
@@ -35,6 +35,16 @@
...
+# Ignore '1.5.0-alpha.*' to '1.5.0a*' messages.
+import warnings
+warnings.filterwarnings('ignore', message='Normalizing .* to .*')
+
+# Ignore 'running' and 'creating' messages
+import logging
+class PyspecFilter(logging.Filter):
+    def filter(self, record):
+        return not record.getMessage().startswith(('running ', 'creating '))
+logging.getLogger().addFilter(PyspecFilter())
```
Comment (ralexstokes) [inline setup.py:47]: I feel like this stuff tends to be fragile so it could more easily break in the future

File: setup.py (hunk 1 above)
Comment (jtraglia) [reply setup.py:397]: I'm not sure. I don't even know how to provide a non-default path. Is this something we do?

File: setup.py (hunk 2 above)
Comment (jtraglia) [reply setup.py:47]: Yeah maybe. Shouldn't be difficult to update if something breaks. Worst case we can just delete it.

File: setup.py (hunk 1 above)
Comment (ralexstokes) [reply setup.py:397]: not sure if its used but it does seem like the code currently supports it

Comment (jtraglia) [reply setup.py:397]: Ah, so it doesn't change any functionality with that. I just removed the print call.

Comment (hwwhww) [reply setup.py:397]: I haven't used the path flags, but Proto did when he designed it: https://github.com/ethereum/consensus-specs/tree/v1.5.0-alpha.6/tests/core/pyspec#advanced

Given that no current use cases require us to *only* build phase0 specs, I think it’s OK to ignore it for now.

**Review by hwwhww — APPROVED** [review]
Looks refreshing. 😎

**Pull request merged**